### ISSUE TITLE
Missing keys

### ISSUE THREAD

**Issue by kcheng1021** [ByteDance-Seed/Depth-Anything-3#2]
Hi, thanks for your great work! I am trying to initialize the mode locally and load your parameters as : 

`
from safetensors.torch import load_file

model = DepthAnything3(model_name="da3-giant")
weight = load_file(model_path)
model.load_state_dict(weight)
`

However, I found the missing keys: "model.head.scratch.output_conv2_aux.1.2.weight", ....

mainly the "model.head.scratch.output_conv2_aux.*"

Does this matter? Or is the parameters and model version are not aligned?

**Comment by haotongl** [comment]
Thank you for your attention. This is not expected, but it currently does not affect the outcome. Try setting model.load_state_dict(weights, strict=False). I am working on checking where the issue is coming from.

**Comment by kcheng1021** [comment]
Thanks for your reply~

**Comment by kcheng1021** [comment]
Reopen to wait for your answer about the issue

**Comment by fangyuedong** [comment]
use_ln = True
        ln_seq = (
            [Permute((0, 2, 3, 1)), nn.LayerNorm(head_features_2), Permute((0, 3, 1, 2))]
            if use_ln
            else []
        )
        self.scratch.output_conv2_aux = nn.ModuleList(
            [
                nn.Sequential(
                    nn.Conv2d(
                        head_features_1 // 2, head_features_2, kernel_size=3, stride=1, padding=1
                    ),
                    *ln_seq,
                    nn.ReLU(inplace=True),
                    nn.Conv2d(head_features_2, 7, kernel_size=1, stride=1, padding=0),
                )
                for _ in range(self.aux_levels)
            ]
        )
seems only **one** nn.LayerNorm(head_features_2) instance is created，the next 3 are just references
